tests/e2e-leg-9/new-online-upgrade-sanity/co-assert.yaml

```diff
@@ -0,0 +1,77 @@
+# (c) Copyright [2021-2024] Open Text.
```

Comment:
What is this for? When will it be verified?

Reply:
Should have ever been committed. Will remove it.